.drone.star

```diff
@@ -14,65 +15,95 @@ windowsglobalimage="cppalliance/dronevs2019"
 
 def main(ctx):
   return [
+  # Priorities: (no 2a, no betas)
+  #
+  # coverage
+  # latest gcc: 11,14,17,20
+  # latest clang: 11,14,17,20
+  # oldest gcc: 4.8
+  # oldest clang: 3.8
+  # asan
+  # tsan
+  # ubsan
+  # valgrind
+  # docs
+  # cmake superproject
+  # cmake install
```

Comment:
cmake and docs should be at the end of this list, under arm/s390x (and update the drone lines)

Reply:
Changed it

(Even though docs fail orders of magnitude more often than arm/s390x and cmake fail orders of magnitude more often than docs)